modules/tinymce/src/plugins/advlist/test/ts/browser/AdvlistOptionsAndToolbarTest.ts

```diff
@@ -85,6 +85,59 @@ describe('browser.tinymce.plugins.advlist.AdvlistOptionsAndToolbarTest', () => {
       })), button);
     });
 
+  context('Context Toolbar Split buttons active state', () => {
+    const hook = TinyHooks.bddSetup<Editor>({
+      ...baseOptions,
+      inline: true, // this is needed because otherwise `TinySelections.setSelection` does not trigger the context toolbar correctly
```

Comment:
Why would that be different I see the bug happening in iframe editors as well? Does it need more waiters?

Reply:
yes the bug is in both, but I'm not able to open the `context toolbar` in tests without `inline: true`, I tried to add waiter etc. but nothing seems to work :/

Reply:
Might be that the editor doesn't have focus. Try adding `,true` to the bddSetup

Reply:
yes, it solves the problem, thanks :)